ajax/libs/blueimp-gallery/package.json

```diff
@@ -36,27 +36,8 @@
     {
       "basePath": "",
       "files": [
-        "css/blueimp-gallery-indicator.css",
-        "css/blueimp-gallery-video.css",
-        "css/blueimp-gallery.css",
-        "css/blueimp-gallery.min.css",
-        "img/error.png",
-        "img/error.svg",
-        "img/play-pause.png",
-        "img/loading.gif",
-        "img/play-pause.svg",
-        "img/video-play.png",
-        "img/video-play.svg",
-        "js/blueimp-gallery-fullscreen.js",
-        "js/blueimp-gallery-indicator.js",
-        "js/blueimp-gallery-video.js",
-        "js/blueimp-gallery-vimeo.js",
-        "js/blueimp-gallery-youtube.js",
-        "js/blueimp-gallery.js",
-        "js/blueimp-gallery.min.js",
-        "js/blueimp-helper.js",
-        "js/jquery.blueimp-gallery.js",
-        "js/jquery.blueimp-gallery.min.js"
+        "img/*",
+        "+(css|js)/*blueimp*.+(css|map|js)"
```

Comment:
`"+(css|js)/**/blueimp*.+(css|map|js)"` would be better.

Reply:
I think that excludes `dist` and `build` as the basePath... we don't have to use `**/*`.
Such as https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/imagehover.css/package.json#L25
https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/css-social-buttons/package.json#L26
Am I wrong?

Thank you.

Reply:
@maruilian11 I think that excludes `dist` and `build` as the basePath... we don't have to use `**/*`.
Such as https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/imagehover.css/package.json#L25
https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/css-social-buttons/package.json#L26
Am I wrong?

Thank you.

Reply:
OK. I think both are fine.